frontend/components/LiveQuery/TargetsInput/TargetsInput.tsx

```diff
@@ -52,7 +50,7 @@ const TargetsInput = ({
 }: ITargetsInputProps): JSX.Element => {
   const dropdownRef = useRef<HTMLDivElement | null>(null);
   const dropdownHosts =
-    searchResults && pullAllBy(searchResults, targetedHosts, "display_name");
+    searchResults && pullAllBy(searchResults, targetedHosts, "id");
```

Comment:
Remind me what this is -- is it fixing a problem where if you had already targeted a host named "xyz", you wouldn't be able to find a different host with the same name in the dropdown?

Reply:
Exactly, since it was defining uniqueness by "display_name"

Reply:
Not explicitly outlined in the bug, but I ran across and fixed